File: projetos_java/idades.java
import java.util.Locale;
import java.util.Scanner;

public class idades {
	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		String nome1, nome2;
		int idade1, idade2;
		double media;
		
		System.out.println("Dados da primeira pessoa:");
		System.out.print("NOME = ");
		nome1 = sc.nextLine();
		System.out.print("IDADE = ");
		idade1 = sc.nextInt();
		
		System.out.println("Dados da segunda pessoa:");
		System.out.print("NOME = ");
		sc.nextLine();
		nome2 = sc.nextLine();
		System.out.print("IDADE = ");
		idade2 = sc.nextInt();
		
		media = (double)(idade1 + idade2) / 2;
		System.out.println("A idade media de" + nome1 + " e " + 
		             nome2 + "e de " + String.format("%.1f", media) + " anos");
		
		
		sc.close();
	}
}

File: projetos_c/idades.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    char    nome1[50], nome2[50];
    int idade1, idade2;
    double  media;

    printf("Dados da primeira pessoa:\n");
    printf("NOME: ");
    gets(nome1);
    printf("IDADES: ");
    scanf("%d", &idade1);

    printf("Dados da segunda pessoa:\n");
    printf("NOME: ");
    fseek(stdin, 0, SEEK_END);
    gets(nome2);
    printf("IDADES: ");
    scanf("%d", &idade2);

    media = (double)(idade1 + idade2) / 2;

    printf("A idade media de %s e %s e de %.1lf anos", nome1, nome2, media);

    return 0;
}
